lib/generators/templates/helpers/visual_spec_helper.tt

```diff
@@ -6,8 +6,7 @@ require_relative 'allure_helper'
 require_relative 'driver_helper'
 require_relative 'visual_helper'
 
-module Raider
-  module SpecHelper
+module SpecHelper
     RSpec.configure do |config|
       config.include(DriverHelper)
       config.include(VisualHelper)
```

Comment:
Just curious. Why are we not including the allure helper here?

Reply:
Because I didn't wanted to make all the allure helper methods available across all tests, this works as a cucumber world that makes the methods available in all the steps